docker-compose/Makefile

```diff
@@ -221,16 +225,13 @@ setup:
 	mkdir -p ~/tmp/openwhisk/minio
 	cp -r ./apigateway/* ~/tmp/openwhisk/api-gateway-config/api-gateway/
 	cp -r ./apigateway/rclone ~/tmp/openwhisk
-       
  
-	$(shell cat ~/tmp/openwhisk/local.env) docker-compose --project-name openwhisk up minio 2>&1 > ~/tmp/openwhisk/setup.log &
```

Comment:
`cat ~/tmp/openwhisk/local.env` is not needed anymore b/c the `.env` file is loaded automatically.